jmx_prometheus_javaagent/src/main/java/io/prometheus/jmx/JavaAgent.java

```diff
@@ -4,18 +4,21 @@
 import io.prometheus.client.hotspot.DefaultExports;
 import java.lang.instrument.Instrumentation;
 import java.io.FileReader;
+import java.util.logging.Logger;
+
 import org.eclipse.jetty.server.Server;
 import org.eclipse.jetty.servlet.ServletContextHandler;
 import org.eclipse.jetty.servlet.ServletHolder;
 import org.eclipse.jetty.util.thread.QueuedThreadPool;
 
 public class JavaAgent {
+   private static final Logger logger = Logger.getLogger(JavaAgent.class.getName());
    static Server server;
 
    public static void premain(String agentArgument, Instrumentation instrumentation) throws Exception {
      String[] args = agentArgument.split(":");
      if (args.length != 2) {
-       System.err.println("Usage: -javaagent:/path/to/JavaAgent.jar=<port>:<yaml configuration file>");
+       logger.severe("Usage: -javaagent:/path/to/JavaAgent.jar=<port>:<yaml configuration file>");
```

Comment:
As this is an error at startup for a command line tool likely being invoked by a human, stderr is appropriate.